pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.snowflake</groupId>
     <artifactId>snowflake-kafka-connector</artifactId>
-    <version>1.6.8</version>
+    <version>1.6.9</version>
```

Comment:
There is some utils.java file as well which uses hardocded 1.6.8, lets fix that too

Reply:
Ah i was looking in xml files only, thanks for the catch!